packages/wallet-standard/src/solana.ts

```diff
@@ -0,0 +1,42 @@
+// This file is copied with modification from @wallet-standard/solana-chains
+
+import { Network } from '@glow-xyz/glow-client';
+import type { IdentifierString } from '@wallet-standard/standard';
+
+/** Solana Mainnet (beta) cluster, e.g. https://api.mainnet-beta.solana.com */
+export const SOLANA_MAINNET_CHAIN = 'solana:mainnet';
+
+/** Solana Devnet cluster, e.g. https://api.devnet.solana.com */
+export const SOLANA_DEVNET_CHAIN = 'solana:devnet';
+
+/** Solana Localnet cluster, e.g. http://localhost:8899 */
+export const SOLANA_LOCALNET_CHAIN = 'solana:localnet';
+
+/** Array of Solana clusters (Glow doesn't support testnet) */
+export const SOLANA_CHAINS = [SOLANA_MAINNET_CHAIN, SOLANA_DEVNET_CHAIN, SOLANA_LOCALNET_CHAIN] as const;
```

Comment:
The chain selection mirrors Glow's `Network` enum.

Reply:
We also don't support localnet so I'll remove that.